src/main/pages/workspace-admin/servers.adoc

```diff
@@ -23,34 +27,39 @@ A server defines protocol port of a process that runs in a machine. It also has
+
 image::workspaces/servers_dashboard.png[]
 
-If your workspace is running, saving a new server will restart a workspace.
+[NOTE]
+====
+If your workspace is running, saving a new server restarts the workspace.
+====
 
 [id="preview-urls"]
 == Preview URLs
 
-Just adding a server with port 3000 does not mean you can use this port to access a server. Each server is assigned with a URL in the runtime, i.e. when a workspace is running. In case of Docker, port 3000 will be published to a random port from the ephemeral port range (32768-65535). In case of OpenShift, a route bound to a service is created. Routes are *persistent URLS*, while server URLS provided in a Docker implementation of Che *change* every time your start a workspace.
+Adding the server with port 3000, does not mean you can use this port to access a server. Each server is assigned a URL when a workspace is running. In the case of Docker, port 3000 is published to a random port from the ephemeral port range from 32768 to 65535. In the case of OpenShift, a route bound to a service is created. Routes are persistent URLS, while the server URLS provided in the Docker implementation of Che change every time you start a workspace.
 
-[id="what-is-my-preview"]
-== What Is My Preview?
+[id="getting-preview-urls"]
+== Getting preview URLs
 
-Say, you have added a server with port 3000 and started a workspace. There are multiple ways to get its preview URL:
+In this example, you added a server with port 3000 and started a workspace. The following are ways to get the server's preview URL:
 
-* Using a link:commands-ide-macro.html[macro] in a command
-* In the IDE, `+` icon in the bottom panel under the editor
+* Use a link:commands-ide-macro.html[macro] a command
+* In the IDE, Click `+` icon in the bottom panel under the editor
 
 image::workspaces/servers.png[]
-* In User Dashboard, *Workspaces > YourWorkspace > Servers* tab
+* In the `User Dashboard`, Click *Workspaces > YourWorkspace > Servers* 
 
-You will also see URLS of some of the internal servers - agents that server launches when the workspace container/pod is up.
+You can also see internal servers URLs.  Internal servers launch when the workspace container or pod is available.
 
-[id="internal-servers"]
-== Internal Servers
+[id="exposing-internal-servers"]
+== Exposing internal servers
 
-Sometimes it is needed to expose port for accessing to process internally withing workspace, but public access should be forbidden. An example of such server may be a database which should be exposed only for web application and because of security concerns it should not be accessible publicly. Then server may be exposed as internal.
+If the port must be accessed internally within the workspace, expose it internally, but do not make it publicly accessible. For example, a database server is exposed only for the web application and because of security concerns it should not be accessible publicly. The database server is exposed as internal.
```

Comment:
```suggestion
To access a port internally within a workspace, expose it internally, but do not make it publicly accessible. For example, a database server is exposed only for the web application and because of security concerns, it should not be accessible publicly. The database server is exposed as internal.
```